<launch>
  <node pkg="frontier_exploration" type="frontier_exploration" name="frontier_exploration" output="screen">

    <param name="num_uavs" value="2" />

    <param name="bandwidth" value="1.0" />
    <param name="lambda" value="0.1" />
    <param name="r_exp" value="1.0" />
    <param name="sensor_range" value="20.0" />
    <param name="waypoint_distance_threshold" value="10.0" />

    <param name="exploration_min_x" value="-52.0" />
    <param name="exploration_max_x" value="0.0" />
    <param name="exploration_min_y" value="-52.0" />
    <param name="exploration_max_y" value="0.0" />
    <param name="exploration_min_z" value="0.0" />
    <param name="exploration_max_z" value="3.0" />

    <param name="just_detection" type="bool" value="true" />
    <param name="comms_range" value="100.0" />
    <param name="min_score" value="1.0" />
    <param name="update_rate" value="1.0" />

  </node>
</launch>